### PR TITLE
Add glossary/helper text to inputs

### PR DESCRIPTION
**Pull request by julia-rom**
trello card [here](https://trello.com/c/n1efESGS/350-glossary-help-for-definitions-tide-current-etc)

summary

Use the `HelperText` for the input components to show additional info about that input (whenever helper text was available).

Helper text was added to :
1. all collect record inputs
2. site
3. management regime

To test:
1. go to a page where helper text is available
2. click on info icon
3. ensure helper text appears and hides when info icon is clicked

### CODE REVIEW

File: src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-import { InputRow, Select, HelperText, RequiredIndicator } from '../../generic/form'
+import { InputRow, Select, HelperText, LabelContainer, RequiredIndicator } from '../../generic/form'
 import { inputOptionsPropTypes } from '../../../library/miscPropTypes'
-
 import InputValidationInfo from '../InputValidationInfo/InputValidationInfo'
 import mermaidInputsPropTypes from '../mermaidInputsPropTypes'
+import { IconButton } from '../../generic/buttons'
+import { IconInfo } from '../../icons'
 
 const InputSelectWithLabelAndValidation = ({
```
Comment (mmnoo) [inline src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js:10]: kinda jokin, kinda not, should we rename this component `InputSelectWithLabelValidationHelperText`?

File: src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js [hunk 2]
```diff
@@ -44,7 +63,7 @@ const InputSelectWithLabelAndValidation = ({
           <option value="">Choose...</option>
           {optionList}
         </Select>
-        {helperText && <HelperText id={`aria-descp${id}`}>{helperText}</HelperText>}
+        {displayHelperText && <HelperText id={`aria-descp${id}`}>{helperText}</HelperText>}
```
Comment (mmnoo) [inline src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js:66]: maybe a ternary for kosherness?

**Review by mmnoo — COMMENTED** [review]
Looks good. Just some comments that are your call, other than the !important one. Just want to check that all the efforts to avoid that were done 

(I didnt QA )

File: src/components/generic/form.js [hunk 3]
```diff
@@ -138,3 +138,7 @@ export const Textarea = styled.textarea`
   resize: vertical;
   ${inputTextareaSelectStyles}
 `
+export const LabelContainer = styled.div`
+  display: flex !important;
```
[the remark this answers is not in the document]
Comment (julia-rom) [reply src/components/generic/form.js:142]: yep normally avoid the `!important,` but in this case it seemed like there was a lot of things overriding it

File: src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js [hunk 4]
```diff
@@ -21,18 +22,36 @@ const InputSelectWithLabelAndValidation = ({
   updateValueAndResetValidationForDuplicateWarning,
   ...restOfProps
 }) => {
+  const [displayHelperText, setDisplayHelperText] = useState(false)
```
[the remark this answers is not in the document]
Comment (julia-rom) [reply src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js:25]: sure I can rename to `isHelperTextShowing`

File: src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js (hunk 1 above)
Comment (julia-rom) [reply src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js:10]: I feel like its a bit unnecessarily verbose (same if we update `InputWithLabelAndValidation`)

File: src/components/generic/form.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (julia-rom) [reply src/components/generic/form.js:142]: we are using grids a lot, which I don't really like using tbh - i'm much more into flexbox, just didn't want to make a bunch of updates for a small piece here

File: src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js (hunk 2 above)
Comment (julia-rom) [reply src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js]: updated!

File: src/components/generic/form.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (mmnoo) [reply src/components/generic/form.js:142]: ok. Ya I dont love grid either TBH.

**Review by mmnoo — APPROVED** [review]

**Pull request merged**